### ISSUE TITLE
Build error in "acor" package: requires global GCC install?

### ISSUE THREAD

**Issue by rossjjennings** [ipta/pulsar-env#6]
In trying to create a copy of the environment on a new machine, I ran into an error installing the `acor` package, which I think is the result of `pip` trying to use a globally-installed version of GCC. I checked and GCC is not installed at the system level on this machine. The text of the error message is below:

```
  × Running setup.py install for acor did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      running install
      /home/ross/anaconda3/envs/IPTA_Env/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-39
      creating build/lib.linux-x86_64-cpython-39/acor
      copying acor/__init__.py -> build/lib.linux-x86_64-cpython-39/acor
      copying acor/acor.py -> build/lib.linux-x86_64-cpython-39/acor
      running build_ext
      building 'acor._acor' extension
      creating build/temp.linux-x86_64-cpython-39
      creating build/temp.linux-x86_64-cpython-39/acor
      gcc -pthread -B /home/ross/anaconda3/envs/IPTA_Env/compiler_compat -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /home/ross/anaconda3/envs/IPTA_Env/include -fPIC -O2 -isystem /home/ross/anaconda3/envs/IPTA_Env/include -fPIC -Iacor -I/home/ross/anaconda3/envs/IPTA_Env/lib/python3.9/site-packages/numpy/core/include -I/home/ross/anaconda3/envs/IPTA_Env/include/python3.9 -c acor/_acor.c -o build/temp.linux-x86_64-cpython-39/acor/_acor.o
      error: command 'gcc' failed: No such file or directory
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> acor

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.

failed

CondaEnvException: Pip failed
```

**Comment by rossjjennings** [comment]
Indeed, an `apt install gcc` fixed this problem. But ideally we shouldn't require users of the environment to have a global GCC install, right?

**Comment by golamshaifullah** [comment]
Could you check that `conda list` doesn't show any gcc packages? It might
be that the acor make script is simply calling for a gcc, instead of the
one in the environment. See
https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html
.

**Comment by rossjjennings** [comment]
@golamshaifullah I'm not sure if I understand what you're saying, but `conda list | grep gcc` gives only
```
_libgcc_mutex             0.1                 conda_forge    conda-forge
libgcc-ng                 12.2.0              h65d4601_19    conda-forge
```

**Comment by golamshaifullah** [comment]
ok. in this case we should simply add gcc to the environment, sinc it looks
like no gcc was installed inside the environment.
I was referring to a known bug where, in most cases package builds in conda
envs search for a local gcc but sometimes old make or cmake scripts will
simply invoke `gcc`, causing builds to look for a system gcc.

**Comment by rossjjennings** [comment]
OK, that makes sense. But in this case it's probably not a conda issue, since acor is being installed with pip, and it's failing during the process of pip building the wheel. Actually, just switching to install acor with conda directly might fix this, since conda is more intelligent about things like build environments. I just checked, and acor 1.1.1 is in conda-forge.

**Comment by JPGlaser** [comment]
I have this switch featured in merge-request #8. That should fix that issue.

~ Joe G.

**Comment by JPGlaser** [comment]
Fixed in #8 which is now merged.
